lib/dotcom_web/components/trip_planner/itinerary_detail.ex

```diff
@@ -19,6 +19,7 @@ defmodule DotcomWeb.Components.TripPlanner.ItineraryDetail do
       ) do
     assigns =
       assign(assigns, :selected_itinerary, Enum.at(itineraries, selected_itinerary_detail_index))
+      |> assign_new(:target, fn -> nil end)
```

Comment:
A test broke without this 🤷🏼‍♀️

Reply:
https://github.com/mbta/dotcom/blob/acb1cbad75bd8e35d46f1a69642125ae480519e3/lib/dotcom_web/components/trip_planner/itinerary_detail.ex#L28 it's still here :/

Reply:
Based on 👇 you should be able to get rid of this now:
- https://github.com/mbta/dotcom/pull/2261

Reply:
Done!